NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -168,9 +188,13 @@ class NeuroID
 
             internal var registeredViews: MutableSet<String> = mutableSetOf()
 
-            internal var endpoint = Constants.productionEndpoint.displayName
+            internal var endpoint = "https://receiver.${Constants.devEndpoint.displayName}"
+            internal var scriptEndpoint = "https://scripts.${Constants.devEndpoint.displayName}"
```

Comment:
According to @priya-xavier, the scripts bucket in PROD is `http://scripts.neuro-id.com/`, not the same as collector. Also isn't this setting everything to dev?

Reply:
oh yeah, I forgot to change this back to prod. ill update it. thanks!

Reply:
separate the script and receiver endpoints in the constants